src/docs/asciidoc/en/disruptor.adoc

```diff
@@ -47,7 +47,7 @@ This is very close to the theoretical limit of a modern processor to exchange da
 
 == Overview
 The Disruptor is the result of our efforts to build the world’s highest performance financial exchange at LMAX.
-Early designs focused on architectures derived from SEDA footnote:SEDA[Staged Event Driven Architecture – http://www.eecs.harvard.edu/~mdw/proj/seda/] and Actors footnote:actors[Actor model – http://dspace.mit.edu/handle/1721.1/6952] using pipelines for throughput.
+Early designs focused on architectures derived from SEDA footnote:SEDA[Staged Event Driven Architecture – https://web.archive.org/web/20160304042910/http://www.eecs.harvard.edu/~mdw/proj/seda/] and Actors footnote:actors[Actor model – https://dspace.mit.edu/handle/1721.1/6952] using pipelines for throughput.
```

Comment:
I would probably link to Wikipedia, or one of its references, to have a good link to SEDA these days e.g. https://en.wikipedia.org/wiki/Staged_event-driven_architecture